src/api/collections-api.tsx

```diff
@@ -0,0 +1,14 @@
+import { getConfig, CollectionsApi, ListCollectionsResponse } from "@imtbl/core-sdk";
```

Comment:
this file should be just `.ts` not `.tsx` as you don't have react components here.